src/components/organisms/UiList/stories/ListBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <UiList
+    v-bind="args"
+    class="list-box"
+  />
+  <div class="search-dropdown-input-row">
+    <UiButton
+      class="search-dropdown-input-row__button ui-button--text"
+      @click="clickHandler"
+    >
+      <UiIcon
+        icon="plus"
+        class="ui-button__icon"
+      />
+      <UiText class="search-dropdown-input-row__button">
```

Comment:
You can put tag `<p>` inside `<button>`: https://www.w3schools.com/tags/tag_button.asp